bin/critterget.py

```diff
@@ -12,14 +12,7 @@
 baseurl = "https://developers.crittercism.com:443/v1.0/"
 authbaseurl = "https://developers.crittercism.com/v1.0/"
 
-# FOR EU PoP USERS ONLY:
-# If you are using Crittercism through its EU data center presence,
-# uncomment the following two lines for baseurl and authbaseurl and 
-# comment out the previous two lines for baseurl and authbaseurl
-# baseurl = "https://developers.eu.crittercism.com:443/v1.0/"
-# authbaseurl = "https://developers.eu.crittercism.com/v1.0/“
```

Comment:
What's different about these comment lines vs those above it on line 9 for instance?  Try adding this comment and see if it breaks something.